nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -711,6 +711,12 @@
 - `python3Packages.nose` has been removed, as it has been deprecated and unmaintained for almost a decade and does not work on Python 3.12.
   Please switch to `pytest` or another test runner/framework.
 
+- `nerdfonts` has been separated into individual font packages under the namespace `nerd-fonts`.
+  The directories for font files have changed from `$out/share/fonts/{opentype,truetype}/NerdFonts`
+  to `$out/share/fonts/{opentype,truetype}/<fontDirName>`, where `<fontDirName>` can be found in
+  the upstream [`fonts.json`](https://github.com/ryanoasis/nerd-fonts/blob/master/bin/scripts/lib/fonts.json)
```

Comment:
Hmm maybe mention https://www.nerdfonts.com/font-downloads instead? It is much easier to view that list v.s reading a JSON file...